### ISSUE TITLE
Allow contentBase for devServer config

### ISSUE THREAD

**Issue by alicoding** [HenrikJoreteg/hjs-webpack#34]
Hi,

I'm currently trying to serve the file from my asset directory, but setting `contentBase` in the config doesn't seem to work and I suspect that it's being overwritten in here everytime:

```
    config.devServer = {
      port: spec.port,
      info: false,
      historyApiFallback: true,
      host: spec.hostname,
      // For some reason simply setting this doesn't seem to be enough
      // which is why we also do the manual entry above and the
      // manual adding of the hot module replacment plugin below
      hot: true
    }
```

I'm doing it using `webpack-dev-server --content-base public` right now, but would love to be able to pass this in in `webpack.config.js` file using 
```
module.exports = getConfig({
  in: './client.jsx',
  out: 'public',
  output: {
    filename: "build/main.js",
  },
  devServer: {
    contentBase: './public'
  },
  hostname: '0.0.0.0',
  port: 3000
}
```

**Comment by alicoding** [comment]
When I console.log `config` in `index.js` file I see `devServer` before:

```
     devServer: { contentBase: '/Users/ali/projects/donate.mozilla.org' },
```

After

```
 devServer:
   { port: 3000,
     info: false,
     historyApiFallback: true,
     host: '0.0.0.0',
     hot: true } }
```

**Comment by lukekarrys** [comment]
Hey @alicoding thanks for the issue!

I just merged in #34 which addresses this and published it to npm as `2.6.0`.

**Comment by alicoding** [comment]
Thanks for fixing that \o/
